Update README.md

lmalenfant opened the pull request:
I added the SonarCloud status to the ReadMe here also.

hayakawa16 commented:
Hi, I not sure if this is related but I just noticed that the readme.md is at the top of the VP splash page above the different repositories.  It just caught my eye because at first I thought the repositories moved because they were below the fold.  Not a problem, just checking.

lmalenfant commented:
Hi @hayakawa16, It is not related to this PR but I told you about the change in the other PR on the VTS repo:

> I added a .github repository in the Virtual Photonics organization so we could put the VP information on the home page. I removed that information here and added the sonar cloud analytics badge to the readme.

hayakawa16 commented:
I see.  Thanks for the explanation.  I had a hunch it was due to the other PR, just checked it after I saw this one.

lmalenfant commented:
@hayakawa16 please could you review this so I can merge? :)

hayakawa16 reviewed (approved):
This looks fine.
